code/game/machinery/requests_console.dm

```diff
@@ -40,15 +41,11 @@ GLOBAL_LIST_EMPTY(allConsoles)
 		// 8 = view messages
 		// 9 = authentication before sending
 		// 10 = send announcement
-	var/silent = 0 // set to 1 for it not to beep all the time
-	var/hackState = 0
-		// 0 = not hacked
-		// 1 = hacked
-	var/announcementConsole = 0
-		// 0 = This console cannot be used to send department announcements
-		// 1 = This console can send department announcements
-	var/open = FALSE // 1 if open
-	var/announceAuth = 0 //Will be set to 1 when you authenticate yourself for announcements
+	var/silent = FALSE // set to 1 for it not to beep all the time
+	var/hackState = FALSE

```

Comment:
not sure about this one

Reply:
hackstate only has two values, TRUE & FALSE. 
![image](https://user-images.githubusercontent.com/6209658/36012764-657027ba-0d2e-11e8-80b8-fc291154240c.png)